datumaro/components/annotation.py

```diff
@@ -147,13 +147,12 @@ def __attrs_post_init__(self):
         self._reindex()
 
     def _reindex(self):
-        indices = {}
+        self._indices = indices = {}
```

Comment:
Please avoid using multiple assignment patterns in a single line of code. There is nothing wrong about it, it just makes the code harder to read and/or understand. It's related to patterns like this one and also `a, b = b, a`, `a = 0; b = 1; c = 2`.